Revit_oM/Parameters/RevitParameter.cs

```diff
@@ -40,6 +40,9 @@ public class RevitParameter : IObject
         [Description("Value of the Revit parameter. Enums are converted to strings, ElementIds to integers.")]
         public virtual object Value { get; set; } = null;
 
+        [Description("Whether the parameter is read only or modifiable by the Revit user.")]
+        public virtual bool IsReadOnly { get; set; } = false;
```

Comment:
I'm just using this property to prevent users from selecting a read-only parameter for value updates. What do you mean by "have one parameter convert plus one set parameter method"?

![image](https://user-images.githubusercontent.com/102604891/177309894-2cd6dc5a-cb90-427d-a91c-0446bc714e0e.png)

Reply:
I've added the integration now. For testing this:
https://github.com/BHoM/Revit_Toolkit/blob/56f784d972dd99c9a958b96cebecb54fd0ec3136/Revit_Core_Engine/Convert/Revit/FromRevit/Parameter.cs#L83
You can use test files ParameterMappingOnPull.gh & ParameterMappingOnPull.rvt in [here](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/03_Alpha/BHoM/Revit_Toolkit/Pull?csf=1&web=1&e=JhQ2H7)https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/03_Alpha/BHoM/Revit_Toolkit/[Pull](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/03_Alpha/BHoM/Revit_Toolkit/Pull?csf=1&web=1&e=JhQ2H7)?csf=1&web=1&e=JhQ2H7.

For testing this:
https://github.com/BHoM/Revit_Toolkit/blob/56f784d972dd99c9a958b96cebecb54fd0ec3136/Revit_Core_Engine/Modify/CopyParameters.cs#L132-L136
You can use files ‘UpdateColumnParameters.gh’ and ‘UpdateColumn.rvt’ in [here](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/03_Alpha/BHoM/Revit_Toolkit/Push?csf=1&web=1&e=1wWH0Y)https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/03_Alpha/BHoM/Revit_Toolkit/[Push](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/12_Scripts/03_Alpha/BHoM/Revit_Toolkit/Push?csf=1&web=1&e=1wWH0Y)?csf=1&web=1&e=1wWH0Y.